src/Vts.Scripting/ForwardSolvers/Demo07PHDOfRhoAndZTwoLayer.cs

```diff
@@ -0,0 +1,71 @@
+﻿namespace Vts.Scripting.ForwardSolvers;
+
+/// <summary>
+/// Class using the Vts.dll library to demonstrate predicting photon hitting density (PHD) for a two-layer medium
+/// as a function of radial extent and depth at a given set of optical properties and source-detector separation
+/// </summary>
+internal class Demo07PhdOfRhoAndZTwoLayer : IDemoScript
+{
+    /// <summary>
+    /// Sample script to demonstrate this class' stated purpose
+    /// </summary>
+    public static void RunDemo(bool showPlots = true)
+    {
+        // Example 07: Compute photon hitting density (PHD) for a two-layer medium as a function
+        // of radial extent and depth at a given set of optical properties and source-detector separation
+
+        // Solver type options:
+        // PointSourceSDA,DistributedGaussianSourceSDA, DistributedPointSourceSDA,
+        // MonteCarlo(basic scaled), Nurbs(scaled with smoothing and adaptive binning)
+        var solver = new TwoLayerSDAForwardSolver { SourceConfiguration = SourceConfiguration.Distributed };
+        var topLayerThickness = 5; // mm
+        var opRegions = new IOpticalPropertyRegion[]
+        {
+            new LayerOpticalPropertyRegion
+            ( 
+                zRange: new DoubleRange(0, topLayerThickness, 2),
+                regionOP: new OpticalProperties(mua: 1, musp: 1, g: 0.8, n: 1.4)
+            ),
+            new LayerOpticalPropertyRegion
+            ( 
+                zRange: new DoubleRange(topLayerThickness, double.PositiveInfinity, 2),
+                regionOP: new OpticalProperties(mua: 0.1, musp: 1, g: 0.8, n: 1.4)
+            )
+        };
+        var rhos = new DoubleRange(start: 0.1, stop: 19.9, number: 100).AsEnumerable().ToArray(); // range of s-d separations in mm
+        var zs = new DoubleRange(start: 0.1, stop: 19.9, number: 100).AsEnumerable().ToArray(); // range of depths in mm
+
+        // predict the tissue's fluence(rho, z) for the given optical properties 
+        //var fluenceOfRhoAndZ = solver.FluenceOfRhoAndZ(new[] { opRegions }, rhos, zs );
```

Comment:
Is this commented code intentional? If so could we explain?

Reply:
Thanks for catching this. You know, I was having trouble with that line compiling, so I reverted to the `ComputationFactory.ComputeFluence` version, and was thinking to add a "todo" to fix/add the overload that would make that code work. But now, as I uncomment it...it works? :\ So I'll go with that instead and adjust the code as needed, accordingly.